databricks/sdk/runtime/__init__.py

```diff
@@ -31,15 +33,22 @@
         _globals[var] = userNamespaceGlobals[var]
     is_local_implementation = False
 except ImportError:
+    from typing import Type, cast
+
     # OSS implementation
     is_local_implementation = True
 
     try:
+        from . import stub
         from .stub import *
+        dbutils_type = Type[stub.dbutils]
     except (ImportError, NameError):
         from databricks.sdk.dbutils import RemoteDbUtils
 
         # this assumes that all environment variables are set
         dbutils = RemoteDbUtils()
+        dbutils_type = RemoteDbUtils
+
```

Comment:
Can you enlighten me if this following branch 
```
    try:
        from . import stub
        from .stub import *
        dbutils_type = Type[stub.dbutils]
```

doesn't throw any exception, then what is `dbutils`? From stub.py it's a class, then why are you casting `dbutils` (class) to `dbutils_type` (Type)?

Reply:
We've defined `dbutils_type` in two places and so it ends up being a Union of the class from the stub and the `ProxyCall` class. We then cast to this Union.

Reply:
Do we really want the union type? It seems like the "RemoteDbutils" type doesn't have useful info for the user, can we just suppress it?